src/components/Listing/Listing.tsx

```diff
@@ -37,7 +37,7 @@ export const Listing = ({
   className,
   variant = 'link_list'
 }: ListingProps) => {
-  const Element = as;
+  const Element = variant === 'link_list' ? 'ul' : as;
```

Comment:
I am making an assumption that we want to create an unordered list. I don't think DCPs have an option to choose the wrapping element and I didn't see `<ol>` anywhere in the codebase.

Reply:
I _think_ it's a safe call but it would be good to test by creating/testing each variant, just in case.